src/_gettsim/transfers/rente.py

```diff
@@ -606,44 +606,151 @@ def ges_rente_frauen_altersgrenze(
     return out
 
 
-def _ges_rente_arbeitsl_altersgrenze(
-    birthdate_decimal: float,
+@dates_active(end="1996-12-31", change_name="_ges_rente_arbeitsl_altersgrenze")
```

Comment:
```suggestion
@dates_active(end="1991-12-31", change_name="_ges_rente_arbeitsl_altersgrenze")
```
Cohorts affected by the law knew from this point onwards what would happen.

Probably means we'll have to introduce another function for 1992-1996?

Same issue below for 2010, or I do not understand the logic here.

Reply:
Yes you are right.
That's actually something I was thinking about this morning: With the current decorator, it is not possible to "reintroduce" a policy, i.e. letting it be active in two different period, right? This applies here because we want to test for Vertrauensschutz, e.g. between 1997 and 2009, but before and after the policy is the same.

Reply:
No, that's not possible given the interface of the decorator. And the case is sufficiently rare as to warrant a (necessarily) more cluttered interface.

In this case it is a one-liner, so the following would be making things more complex, don't use. But you could have a common function that is called under the hood (it would not be visible in the DAG). Would make it clear that it is the same regime.

Reply:
What you did works perfectly well and is even better than what I had in mind!